index.html

```diff
@@ -2,7 +2,153 @@
 <html>
   <head>
     <title>Memory exercise</title>
+	<script src="https://ajax.googleapis.com/ajax/libs/jquery/1.12.0/jquery.min.js"></script>
+
+	<style>
+		.memoryboard {
+			background-color: #CCC; 
+			
+		}
+
+		.memoryboard > div {
+			width: 71px; 
+			height: 71px; 
+		}
+
+		.flipped { 
+			background-color: white !important; 
+		}
+
+		.tile {
+			display: inline-block; 
+			background-color: black; 
+			margin: 10px; 
+		}
+	</style>
+
   </head>
   <body>
+	  <h1>memory</h1>
+	<div class = "memoryboard">
+	</div>
+  	<div class = "msg"> pick a card</div>
+  	
+  	<script>	
+  	var Tile = function(text) {
+		this.text = text 
+		this.div =  $("<div class=\"tile\"><span>" + this.text +"</span></div>");
+		console.log("text", this.text)
+		$(".memoryboard").append(this.div)
+		
+		var self = this 		
+		this.div.click(function(){
+			self.handleClick();
+		})
```

Comment:
Yes, I sort of remember talking about this in class. I also just watched a tutorial about factory functions and how to avoid using "this", so trying to process that too, hehe.